PSMDB-1372 Pro builds documentation (7.0)

nastena1606 opened the pull request:
modified:   .gitignore
	modified:   docs/404.md
	new file:   docs/fips.md
	modified:   docs/install/index.md
	new file:   docs/install/install-pro.md
	modified:   docs/install/source.md
	new file:   docs/install/update-pro.md
	modified:   docs/install/upgrade-from-mongodb.md
	new file:   docs/psmdb-pro.md
	modified:   mkdocs-base.yml

nastena1606 committed:
PSMDB-1372 Pro builds documentation (7.0)

modified:   .gitignore
	modified:   docs/404.md
	new file:   docs/fips.md
	modified:   docs/install/index.md
	new file:   docs/install/install-pro.md
	modified:   docs/install/source.md
	new file:   docs/install/update-pro.md
	modified:   docs/install/upgrade-from-mongodb.md
	new file:   docs/psmdb-pro.md
	modified:   mkdocs-base.yml

janwieremjewicz reviewed (changes requested):
We should refrain from using PSMDB abbreviation as it's not an officially approved one. 
Following the discussions regarding "PDMDB Pro" or "PSDMB Professional" let's follow the pattern of "Pro builds" or "Professional builds". The "Pro" refers to the build not to the code as code is open source.

nastena1606 committed:
Changed naming by adding builds

modified:   docs/fips.md
	modified:   docs/install/install-pro.md
	modified:   docs/install/source.md
	modified:   docs/install/update-pro.md
	modified:   docs/psmdb-pro.md
	modified:   mkdocs-base.yml

nastena1606 committed:
Implemented feedback from marketing

nastena1606 committed:
Merge branch '7.0' into PSMDB-1372-Pro-build-docs-7.0
